go.mod

```diff
@@ -27,15 +27,15 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.26.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20240618051212-6a5ade3468a9
+	github.com/percona/everest-operator v0.6.0-dev1.0.20240714061521-01ec1d93af76
 	github.com/rodaine/table v1.2.0
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/viper v1.18.2
 	github.com/stretchr/testify v1.9.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.25.0
-	golang.org/x/mod v0.18.0
-	golang.org/x/net v0.27.0
+	golang.org/x/crypto v0.24.0
+	golang.org/x/mod v0.19.0
+	golang.org/x/net v0.26.0
```

Comment:
No idea, I didn't notice.. I just ran go mod tidy.. Will reverse it

Reply:
oh, never mind. Its mostly because I've changed the everest-operator package to `EVEREST-1247`  to be able to use the `GetNextUpgrade` function. It will be reverted back once I merged the everest-operator PR and clean-up go mod finally..